src/Repository/index.js

```diff
@@ -83,21 +83,13 @@ class Repository {
     async createBranch(branchName) {
         const fromSha = await this.getHeadRef(this.defaultBranch)
 
-        try {
-            // https://octokit.github.io/rest.js/#api-Git-createRef
-            await this.github.git.createRef({
-                owner: this.owner,
-                repo: this.repo,
-                ref: `refs/heads/${branchName}`,
-                sha: fromSha,
-            })
-
-            return false
-        } catch (error) {
-            // branch already exists
```

Comment:
What would be the advantage and reason for not handling situations where `this.github.git.createRef` fails and throws an error?